spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -290,13 +290,12 @@
       before do
         @registration = create(:registration,
                                conference: conference,
-                               user:       user,
-                               arrival:    Date.new(2014, 04, 25))
+                               user:       user)
       end
 
       context 'updates successfully' do
         before do
-          patch :update, params: { registration:  attributes_for(:registration, arrival: Date.new(2014, 04, 29)),
+          patch :update, params: { registration:  attributes_for(:registration),
```

Comment:
I think we should either give it other parameters or drop this test. This is not really testing something.

Reply:
Added a different (random) parameter.